services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -231,6 +247,7 @@ const RenderEMAPermitCommonSections = ({ code, isAmendment, index, isDisabled })
         documents={tableDocuments}
         documentParent="project summary authorization"
         documentColumns={documentColumns}
+        removeDocument={!isDisabled && isEditMode ? onDeleteDocument : undefined}
```

Comment:
not sure if here we want to use isDisabled or if we want to use docFieldsDisabled (as above in `AuthorizationSupportDocumentUpload`). See ticket 6129

Reply:
I guess I need to confirm what state's document deletion should be allowed in. docFieldsDisabled is still enabled even if a project is no longer in draft which is why I have been using the fieldsDisabled flag instead. (isDisabled = fieldsDisabled on line 660).